web/cypress/fixtures/plugin-dataset.json

```diff
@@ -463,7 +463,6 @@
         "conn": 1,
         "burst": 0,
         "default_conn_delay": 0.1,
-        "rejected_code": 503,
```

Comment:
hi @bzp2010 I noticed that `rejected_code` is still one of the properties in `limit-conn` plugin with default value 503, may I ask why remove it here?

Reply:
I think it has default values, so just test the use cases where it has outliers.  check L600

Reply:
got it, thanks for your explanation